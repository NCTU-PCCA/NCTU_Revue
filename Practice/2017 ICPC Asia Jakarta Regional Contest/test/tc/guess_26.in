858073546874608830 36
>><>><>>><<<<<>><<<<<<><<>><>><>>><<
