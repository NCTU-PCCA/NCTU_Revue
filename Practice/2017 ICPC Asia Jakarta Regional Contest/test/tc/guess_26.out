1 2 858073546874608830 3 4 858073546874608829 5 6 7 858073546874608828 858073546874608827 858073546874608826 858073546874608825 858073546874608824 8 9 858073546874608823 858073546874608822 858073546874608821 858073546874608820 858073546874608819 858073546874608818 10 858073546874608817 858073546874608816 11 12 858073546874608815 13 14 858073546874608814 15 16 17 858073546874608813 858073546874608812
